python/nano/src/bigdl/nano/utils/inference/pytorch/model_utils.py

```diff
@@ -82,3 +84,37 @@ def export_to_onnx(model, input_sample=None, onnx_path="model.onnx", dynamic_axe
                                 }
     default_onnx_export_args.update(kwargs)
     torch.onnx.export(model, input_sample, onnx_path, **default_onnx_export_args)
+
+
+class FP32InferenceModelWrapped(pl.LightningModule):
```

Comment:
we should also explicitly make this class inference only (e.g., overriding `train`)

Reply:
Since it's only internal. I will not override `train`.